src/embed/App.js

```diff
@@ -103,14 +106,12 @@ export default class App extends React.PureComponent {
         .then(res => res.json())
         .then(camelizeKeys);
 
-      this.setState({
-        sandbox: response.data,
-        currentModule:
-          this.state.currentModule ||
+      const currentModule = this.state.currentModule ||
           response.data.modules.find(
             m => m.title === 'index.js' && m.directoryShortid == null,
```

Comment:
Hmm, not part of the review, but maybe we could change this to `findMainModule` from the modules selector. Seems like I did a quick hack here.